integration/integration_test.go

```diff
@@ -51,6 +51,14 @@ func TestLifecycle(t *testing.T) {
 	)
 }
 
+func TestCmdsCreateProjectTomlFilesIfNonexistent(t *testing.T) {
```

Comment:
Put this in `common/projectconfig/integration_test.go` now that we can run integration tests from anywhere.

Reply:
niiiiiice! moved